src/components/device-page/DeviceSettings.tsx

```diff
@@ -54,14 +54,40 @@ export class DeviceSettings extends Component<DeviceSettingsProps, DeviceSetting
     render(): ReactNode {
         const { schema, data, uiSchema } = this.getSchemaAndConfig();
         return (
-            <Form
-                schema={schema}
-                formData={data}
-                onSubmit={this.updateConfig}
-                onChange={this.onFormChange}
-                uiSchema={uiSchema}
-                fields={{ TitleField, DescriptionField }}
-            />
+            <div>
```

Comment:
I will humbly listen to suggestions on how to refactor this to fix https://codebeat.co/projects/github-com-nurikk-zigbee2mqtt-frontend-dev/pull_requests/734366

Anything similar in the source code, that I can look at for inspiration? I'm pretty new to React ;)

Reply:
There's already libraries to handle icons
e.g.

```
<i class="fa-solid fa-circle-info"></i>
```
https://fontawesome.com/icons/circle-info?f=classic&s=solid

Reply:
https://www.npmjs.com/package/@fortawesome/fontawesome-free

Reply:
https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/2e2272f9ac9d7894f2311e8ee6867af510b8caa5

Reply:
Unfortunately, the function is still "too long": https://codebeat.co/projects/github-com-nurikk-zigbee2mqtt-frontend-dev/pull_requests/734366

Reply:
I could just get rid of some whitespace, but I'd rather go into a more proper direction, like extracting something like `<InfoAlert message="blahblah" />`

Reply:
https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/06b2fdae70352b05fda115a3627abc685d92ce0c